sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/feedranges/FeedRangeEpkImpl.java

```diff
@@ -249,4 +249,4 @@ public void removeProperties(JsonSerializable serializable) {
         checkNotNull(serializable, "Argument 'serializable' must not be null.");
         serializable.remove(Constants.Properties.RANGE);
     }
-}
\ No newline at end of file
+}
```

Comment:
nit: revert this file.

Reply:
It keeps complaining everytime about "No new line at the end of the file", so I will keep this change to get rid of that.